package.json

```diff
@@ -10,7 +10,7 @@
     "tsc-watch": "tsc -w",
     "webpack-watch": "webpack -w",
     "clean": "rm -rf built",
-    "publish": "tsc && webpack --config webpack.production.config.js",
+    "prepublish": "tsc && webpack --config webpack.production.config.js",
```

Comment:
happy that you renamed this. Easy to confuse "npm publish" with "npm run publish"